### ISSUE TITLE
geoip returns a `null` value

### ISSUE THREAD

**Issue by TarekRaafat** [yusukeshibata/express-fingerprint#14]
`geoip` returns a `null` value.

When I bypass the `_geoipLite2` function and use the below instead it works as expected.

> Doesn't work
```js
const geo = _geoipLite2.default.lookup(this.req.ip);
```

> Works
```js
const geo = _geoipLite.lookup(this.req.ip);
```

Any idea why this is occurring?

**Comment by italofialho** [comment]
Same here :(

**Comment by victor-gabou** [comment]
+1

**Comment by yusukeshib** [comment]
@TarekRaafat
@italofialho
@victor-gabou

I released v1.2.0, I think it's fixed.
